marketplacetests/marketplace_gaia_test.py

```diff
@@ -29,5 +29,5 @@ def install_certs(self):
             self.device.push_file(os.path.join(certs_folder, file_name),
                                   destination='data/b2g/mozilla/%s/%s' % (profile_folder, file_name))
         self.data_layer.set_char_pref('dom.mozApps.signed_apps_installable_from',
-                                      'https://marketplace-dev.allizom.org,https://marketplace.firefox.com')
+                                      'https://marketplace-dev.allizom.org,https://marketplace.firefox.com,https://marketplace.allizom.org')
```

Comment:
I added this because in-app tester App was only added on stage and then added on dev and I forgot about it.
I will remove it.